### DISCUSSION TITLE
tailwind does not load css for static pages

### DISCUSSION THREAD

**Discussion by gabriel-logan** [vercel/next.js#63784] [Help] [answered]
### Summary

When I build with the output set to export tailwind it doesn't load the css for the static pages, I wanted to create a static website, I'm using next with tailwind, in dev mode everything works great, when I run yarn start everything also works perfect, with styling etc. .

But the /out folder where there are static files, when I run it in my Apache it opens without any css, without any styling.

someone help me ?

**Comment by gabriel-logan** [comment, marked as answer]
Solved, I just needed to pass the assetPrefix and the
basePath to my build output folder

like this 

/** @type {import('next').NextConfig} */
const nextConfig = {
	output: 'export',
	assetPrefix: '/out',
	basePath: '/out',
};

export default nextConfig;

**Reply by gabriel-logan** [reply]
If you need to change the name of the output folder

use distDir: 'nameyouchoose', but remember to change the path and assets

**Reply by parashgoswami** [reply]
I solve my problem. Thank you
But my images are not rendered. If remove "/" from the src then it works
